### PR TITLE
Set welcome message on subscription metadata

### PR DESCRIPTION
**Pull request by imsickofmaps**
see https://github.com/praekelt/seed-stage-based-messaging/issues/9

### CODE REVIEW

File: registrations/tests.py [hunk 1]
```diff
@@ -1111,6 +1113,8 @@ def test_friend_only_voice_mon_wed_2_5(self):
         self.assertEqual(d_friend.next_sequence_number, 5)
         self.assertEqual(d_friend.lang, "eng_NG")
         self.assertEqual(d_friend.schedule, 3)
+        self.assertEqual(d_friend.metadata["prepend_next_delivery"],
```
Comment (imsickofmaps) [inline registrations/tests.py:1197]: @gsvr not sure about this one

**Comment by imsickofmaps** [comment]
@gsvr review please

**Comment by imsickofmaps** [comment]
@gsvr I've just realised I've sent the SMS as a prepend too instead of immediately as per https://github.com/praekelt/hellomama-registration/issues/36 - will resolve tomorrow, please review the rest though.

File: registrations/tests.py (hunk 1 above)
Comment (gsvr) [inline registrations/tests.py:1197]: I'm actually not sure either. I vaguely recall seeing a note somewhere that for this kind of registration where the same user gets the mother and household messages, that we should send only one welcome message (probably the mother's).  But I can't find it in a scope now, so I'm unsure.

**Comment by gsvr** [comment]
I'm happy with it in principle.  If you want to you can land this and then we can change the welcome SMS sending in #36, otherwise we could do the work for #36 here.

**Comment by imsickofmaps** [comment]
@gsvr ready for re-review

**Comment by gsvr** [comment]
:+1: when travis happy

**Pull request merged**